scalapack symbols check is broken

https://github.com/dealii/dealii/pull/7820 does not work in my environment.

```
CMake Error at cmake/macros/macro_configure_feature.cmake:112 (MESSAGE):
  

  Could not find the scalapack library!

  Could not find a sufficient SCALAPACK installation:

  SCALAPACK symbol check for pdsyevr_ and pssyevr failed! This usually means
  that your SCALAPACK installation is incomplete or the link line is broken.
  Consult

    CMakeFiles/CMakeError.log

  for further information.
```

with `CMakeError.log`

```
Performing C SOURCE FILE Test DEAL_II_SCALAPACK_HAS_PDSYEVR_PSSYEVR failed with the following output:
Change Dir: /home/davydden/Desktop/work/C++/dealii/_build/CMakeFiles/CMakeTmp

Run Build Command:"/usr/bin/make" "cmTC_f4945/fast"
/usr/bin/make -f CMakeFiles/cmTC_f4945.dir/build.make CMakeFiles/cmTC_f4945.dir/build
make[1]: Entering directory '/home/davydden/Desktop/work/C++/dealii/_build/CMakeFiles/CMakeTmp'
Building C object CMakeFiles/cmTC_f4945.dir/src.c.o
/home/davydden/spack/opt/spack/linux-ubuntu18.10-x86_64/gcc-8.2.0/openmpi-3.1.3-srvcccqgldcupnqkneyf5ufwqd2adti6/bin/mpicc   -DDEAL_II_SCALAPACK_HAS_PDSYEVR_PSSYEVR   -o CMakeFiles/cmTC_f4945.dir/src.c.o   -c /home/davydden/Desktop/work/C++/dealii/_build/CMakeFiles/CMakeTmp/src.c
Linking C executable cmTC_f4945
/home/davydden/spack/opt/spack/linux-ubuntu18.10-x86_64/gcc-8.2.0/cmake-3.13.4-v7nrhpsovaqrjsmbhpbo4qx3rtrfgtw4/bin/cmake -E cmake_link_script CMakeFiles/cmTC_f4945.dir/link.txt --verbose=1
/home/davydden/spack/opt/spack/linux-ubuntu18.10-x86_64/gcc-8.2.0/openmpi-3.1.3-srvcccqgldcupnqkneyf5ufwqd2adti6/bin/mpicc  -DDEAL_II_SCALAPACK_HAS_PDSYEVR_PSSYEVR    -rdynamic CMakeFiles/cmTC_f4945.dir/src.c.o  -o cmTC_f4945 
/usr/bin/ld: CMakeFiles/cmTC_f4945.dir/src.c.o: in function `main':
src.c:(.text+0xa): undefined reference to `pdsyevr_'
/usr/bin/ld: src.c:(.text+0x14): undefined reference to `pssyevr_'
collect2: error: ld returned 1 exit status
make[1]: *** [CMakeFiles/cmTC_f4945.dir/build.make:87: cmTC_f4945] Error 1
make[1]: Leaving directory '/home/davydden/Desktop/work/C++/dealii/_build/CMakeFiles/CMakeTmp'
make: *** [Makefile:121: cmTC_f4945/fast] Error 2

Source file was:

        void pdsyevr_();
        void pssyevr_();
        int main(){
          pdsyevr_();
          pssyevr_();
          return 0;
        }
```
I don't see what it tries to link against, though...

my scalapack setup is 
```
SCALAPACK_INCLUDE_DIRS = /home/davydden/spack/opt/spack/linux-ubuntu18.10-x86_64/gcc-8.2.0/netlib-scalapack-2.0.2-c4oh6ijycn3ghtybd3xafasg57ahmb3c/include
#            SCALAPACK_LIBRARIES = /home/davydden/spack/opt/spack/linux-ubuntu18.10-x86_64/gcc-8.2.0/netlib-scalapack-2.0.2-c4oh6ijycn3ghtybd3xafasg57ahmb3c/lib/libscalapack.so
```
and

```
$ nm /home/davydden/spack/opt/spack/linux-ubuntu18.10-x86_64/gcc-8.2.0/netlib-scalapack-2.0.2-c4oh6ijycn3ghtybd3xafasg57ahmb3c/lib/libscalapack.so | grep pdsyevr
00000000003d4370 T pdsyevr_
```


p.s.  I do not wipe build folder, but I also don't see how this can influence the check.

--- a/cmake/configure/configure_scalapack.cmake
+++ b/cmake/configure/configure_scalapack.cmake
@@ -28,7 +28,7 @@ MACRO(FEATURE_SCALAPACK_FIND_EXTERNAL var)
     CHECK_MPI_INTERFACE(SCALAPACK ${var})
 
     IF (${var})
-      SET(CMAKE_REQUIRED_LIBRARIES ${SCALAPACK_LIBRARY})
+      SET(CMAKE_REQUIRED_LIBRARIES ${SCALAPACK_LIBRARIES})
       CHECK_C_SOURCE_COMPILES("
         void pdsyevr_();
         void pssyevr_();
@@ -47,7 +47,7 @@ MACRO(FEATURE_SCALAPACK_FIND_EXTERNAL var)
         SET(SCALAPACK_ADDITIONAL_ERROR_STRING
           ${SCALAPACK_ADDITIONAL_ERROR_STRING}
           "Could not find a sufficient SCALAPACK installation: \n"
-          "SCALAPACK symbol check for pdsyevr_ and pssyevr failed! "
+          "SCALAPACK symbol check for pdsyevr_ and pssyevr_ failed! "
           "This usually means that your SCALAPACK installation is incomplete "
           "or the link line is broken. Consult\n"
           "  CMakeFiles/CMakeError.log\n"

